[New Product] DuckDb

adriens opened the pull request:
# :grey_question: About

[since this week](https://x.com/duckdb/status/1967920610874707986), 

> 📈 DuckDB 1.4.0 is out! This is our first LTS release which comes with *one year of community support*. It also supports database encryption, the MERGE SQL statement and Iceberg writes.

<img width="603" height="676" alt="image" src="https://github.com/user-attachments/assets/ec12960e-4bb0-4380-ba07-5ee77dbc6da8" />

Also, cf releaseNote, it starts to [Long Term Support (LTS) Edition](https://duckdb.org/2025/09/16/announcing-duckdb-140#long-term-support-lts-edition) : 

<img width="1041" height="606" alt="image" src="https://github.com/user-attachments/assets/7447eca2-fff8-4f85-bbe7-c241e170b16c" />

# :bookmark_tabs: Resources

- [Tweet](https://x.com/duckdb/status/1967920610874707986)
- [Announcing DuckDB 1.4.0 LTS
](https://duckdb.org/2025/09/16/announcing-duckdb-140#long-term-support-lts-edition)

adriens committed:
[New Product] `DuckDb`

adriens committed:
Update duckdb.md

marcwrobel commented on products/duckdb.md:
```diff
@@ -0,0 +1,63 @@
+iconSlug: duckdb
+permalink: /duckdb
+alternate_urls:
+  - /duck-db
```
```suggestion
```

[Could not find](https://www.google.com/search?q=%22duck-db%22) any reference of duckdb under this name.

marcwrobel commented on products/duckdb.md:
```diff
@@ -0,0 +1,63 @@
+alternate_urls:
+  - /duck-db
+versionCommand: duckdb version
+releasePolicyLink: https://duckdb.org/2025/09/16/announcing-duckdb-140
```
```suggestion
releasePolicyLink: https://duckdb.org/2025/09/16/announcing-duckdb-140.html#long-term-support-lts-edition
```

marcwrobel commented on products/duckdb.md:
```diff
@@ -0,0 +1,63 @@
+  - /duck-db
+versionCommand: duckdb version
+releasePolicyLink: https://duckdb.org/2025/09/16/announcing-duckdb-140
+changelogTemplate: https://github.com/duckdb/duckdb/releases/tag/__LATEST__
```
```suggestion
changelogTemplate: https://github.com/duckdb/duckdb/releases/tag/v__LATEST__
```

marcwrobel commented on products/duckdb.md:
```diff
@@ -0,0 +1,63 @@
+
+identifiers:
+  - repology: duckdb
+
```
Automation based on git tags (`git`) is possible. See https://github.com/endoflife-date/endoflife.date/blob/adffaa1050be03c3d9cc4401e143adc248d76b99/products/apache-couchdb.md?plain=1#L22 for example.

adriens replied:
I've just added it, thanks for the example @marcwrobel :pray:

marcwrobel commented on products/duckdb.md:
```diff
@@ -0,0 +1,63 @@
+
+releases:
+  - releaseCycle: "1.4"
+    releaseDate: 2025-09-16
```
```suggestion
    releaseDate: 2025-09-16
    codename: "Andium"
```

Codename could be added on all releases.

marcwrobel replied:
On all releases please.

adriens replied:
First time I see this possibility, thanks a lot for pointing it @marcwrobel :rocket:

marcwrobel commented on products/duckdb.md:
```diff
@@ -0,0 +1,63 @@
+For LTS DuckDB versions, community support will last a year after the release (for now). DuckDB Labs is also starting
+to offer support for older LTS versions after their community support has expired.
+This doesn't mean they are supporting [all SQLite releases](https://www.sqlite.org/chronology.html);
+it means they plan as if they will be supporting SQLite until 2050.
```
Why mentioning SQLite ?

adriens replied:
That's a typo and a bad copy/paste (I took sqlit as a bassis for this PR)

adriens replied:
Just removed the lines, I did not see them at the very bottom of the `md`

marcwrobel commented on products/duckdb.md:
```diff
@@ -0,0 +1,63 @@
+
+> [DuckDB](https://duckdb.org/) is an analytical in-process, serverless SQL database management system.
+
+Starting with `v1.4.0`, every other DuckDB version is going to be a Long Term Support (LTS) edition.
```
What about non-LTS releases ?

adriens replied:
I understand that for non-LTS ones, only the latest are supported : what's your understanding @marcwrobel ?

marcwrobel replied:
That's what they say in  https://duckdb.org/2025/09/16/announcing-duckdb-140.html#long-term-support-lts-edition :+1: 

> In the past, we would automatically deprecate old DuckDB versions whenever the newer version was released. But we’re changing this today.

marcwrobel replied:
Non-LTS releases support policy is still not documented.

marcwrobel reviewed (changes requested):
Thanks @adriens, here are a few notes and suggestions.

adriens committed:
Update products/duckdb.md

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

adriens committed:
Update products/duckdb.md

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

adriens committed:
Update products/duckdb.md

Thanks Marc

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

adriens committed:
Update products/duckdb.md

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

adriens committed:
Merge branch 'master' into patch-1

adriens committed:
Update duckdb.md > Remove sqlite mentions

as spotted in 
- https://github.com/endoflife-date/endoflife.date/pull/8515#discussion_r2376477881

adriens committed:
Update products/duckdb.md

Co-authored-by: Marc Wrobel <marc.wrobel@gmail.com>

adriens committed:
Merge branch 'master' into patch-1

adriens committed:
Update duckdb.md > Add codename

adriens committed:
Update duckdb.md > Remove spaces at end of file

adriens committed:
Update duckdb.md

adriens committed:
Merge branch 'master' into patch-1

adriens committed:
Update duckdb.md

Add `auto` cf
- https://github.com/endoflife-date/endoflife.date/pull/8515#discussion_r2376465137

adriens committed:
Update duckdb.md

https://duckdb.org/2025/10/07/announcing-duckdb-141

adriens committed:
Merge branch 'master' into patch-1

adriens committed:
Merge branch 'master' into patch-1

adriens committed:
Update duckdb.md

cf
- https://github.com/endoflife-date/endoflife.date/pull/8515#issuecomment-3395338571

marcwrobel committed:
Revise DuckDB release and support information

Updated release policy and support details for DuckDB.

marcwrobel commented on products/duckdb.md:
```diff
@@ -0,0 +1,70 @@
+LTS releases are supported for one year.
+Non-LTS releases become end-of-life once a newer release (LTS or not) is available.
+
+[Commercial support by DuckDB Labs](https://duckdblabs.com/) is also available older LTS versions after their community support has expired.
```
```suggestion
[Commercial support by DuckDB Labs](https://duckdblabs.com/) is also available for older LTS releases after their community support has expired.
```

marcwrobel committed:
Update products/duckdb.md

marcwrobel committed:
Update addedAt date for DuckDB documentation

marcwrobel committed:
Update DuckDB release information and methods
